### ISSUE TITLE
"Package sensio/framework-extra-bundle is abandoned, you should avoid using it. Use Symfony instead."

### ISSUE THREAD

**Issue by Leont** [roodjong/mijnrood#119]
Tijdens het installeren van mijnrood krijg ik de waarschuwing "Package sensio/framework-extra-bundle is abandoned, you should avoid using it. Use Symfony instead.".

Mogelijk is issue #118 hierdoor veroorzaakt?

**Comment by Xesau** [comment]
Nee, heeft niets met #118 te maken. Deze bundle moeten we te zijner tijd eens vervangen met de nieuwe functies in Symfony 5/6
